src/error-display/error-display-data.tsx

```diff
@@ -235,3 +236,133 @@ export const ERROR_DISPLAY_DATA = new Map<
+                    correct.
+                </>
+            ),
+        },
+    ],
+    [
+        "408",
+        {
+            img: oldImgAttributeHelper(ImgPaths["400"]),
+            title: "Something went wrong",
+            description: (
+                <>
+                    Please refresh the page or try again later. If it&rsquo;s
+                    still not working, email{" "}
+                    <a href="mailto:helpdesk@life.gov.sg">
+                        helpdesk@life.gov.sg
+                    </a>{" "}
+                    and let us know you received a HTTP 408 error.
+                </>
+            ),
+        },
+    ],
+    [
+        "500",
+        {
+            img: imgAttributeHelper(ImgPaths["500"]),
+            title: "Something went wrong",
+            description: (
+                <>
+                    This could be a temporary problem, so please refresh the
+                    page or try again later.
+                </>
+            ),
+            renderDescription: (attrs: Error500AdditionalAttributes) => (
+                <>
+                    This could be a temporary problem, so please refresh the
+                    page or try again later.
+                </>
+            ),
+        },
+    ],
+    [
+        "503",
+        {
+            img: oldImgAttributeHelper(ImgPaths["503"]),
+            title: "Service under maintenance",
+            description:
+                "This service is currently unavailable. Please try again later.",
+        },
+    ],
+    [
+        "maintenance",
+        {
+            img: oldImgAttributeHelper(ImgPaths["503"]),
+            title: "Service under maintenance",
+            description:
+                "This service is currently unavailable. Please try again later.",
+            renderDescription: (attrs: MaintenanceAdditionalAttributes) => (
+                <>
+                    This service is currently unavailable. Please try again
+                    after&nbsp;
+                    <strong>{attrs.dateString}</strong>.
+                </>
+            ),
+        },
+    ],
+    [
+        "unsupported-browser",
+        {
+            img: imgAttributeHelper(ImgPaths["unsupported-browser"]),
+            title: "Browser not supported",
+            description:
+                "To use BookingSG, download the latest version of Chrome, Edge, Firefox or Safari.",
+        },
+    ],
+    [
+        "partially-supported-browser",
+        {
+            img: imgAttributeHelper(ImgPaths["unsupported-browser"]),
+            title: "Browser not supported",
+            description:
+                "BookingSG works best with the latest version of Chrome, Edge, Firefox or Safari. If you continue with your browser, you may run into problems.",
+        },
+    ],
+]);
+
+export const errorDisplayData = (
+    type: ErrorDisplayType,
+    colorScheme: ColorScheme = "base"
```

Comment:
`colorScheme` is not a correct key to use in this case. Pending discussion outcome